services/core-web/src/components/mine/Projects/ProjectSummary.tsx

```diff
@@ -109,12 +107,11 @@ export const ProjectSummary: FC = () => {
           setIsValid(false);
         });
     }
-    return dispatch(fetchMineRecordById(projectMineGuid)).then(() => {
-      const mine = mines[projectMineGuid];
+    return dispatch(fetchMineRecordById(projectMineGuid)).then((res) => {
```

Comment:
this made cypress fail, so I tested it out locally and it failed locally too. 🤷‍♀️ 
Race condition- it's not (always) finished populating the redux state immediately in the .then so you get undefined errors.

Reply:
technically it probably shouldn't be fetching the mine record unless it doesn't already exist in the state (which it will if you navigated to the page in most ways aside from directly loading the page). But this whole page needs an overhaul and this is not the ticket.